test/e2e/nodenetworkpolicy_test.go

```diff
@@ -891,3 +898,101 @@ func testNodeACNPNestedIPBlockClusterGroupCreateAndUpdate(t *testing.T) {
+	// In a Kind cluster, each Kubernetes Node runs as a Docker container. The iptables LOG target doesn't generate
+	// kernel logs inside containers unless /proc/sys/net/netfilter/nf_log_all_netns is set to 1. However, this setting
+	// cannot be modified in GitHub Actions.
+	skipIfProviderIs(t, "kind", "The iptables LOG action doesn't generate log in container")
+
+	randomLogLabel := randSeq(12) // Generate a random logLabel with 12 characters long.
+	toBeTruncated := "thisPartShouldBeTruncated"
+	logLabel := randomLogLabel + toBeTruncated // Append the suffix to the logLabel.
+	builder := &ClusterNetworkPolicySpecBuilder{}
+	builder = builder.SetName("acnp-drop-x-to-y-egress").
+		SetPriority(1.0).
+		SetAppliedToGroup([]ACNPAppliedToSpec{{NodeSelector: map[string]string{labelNodeHostname: nodes["x"]}}})
+	builder.AddEgress(ProtocolTCP, &p80, nil, nil, nil, nil, nil, nil, nil, nil, map[string]string{labelNodeHostname: nodes["y"]}, nil,
+		nil, nil, nil, nil, nil, crdv1beta1.RuleActionDrop, "", "", nil)
+	builder.AddEgressLogging(logLabel)
+
+	acnp, err := k8sUtils.CreateOrUpdateACNP(builder.Get())
+	require.NoError(t, err)
+	require.NoError(t, data.waitForACNPRealized(t, acnp.Name, policyRealizedTimeout))
+
+	antreaPodName, err := data.getAntreaPodOnNode(nodes["x"])
+	require.NoError(t, err)
+
+	expectedLogPrefix := fmt.Sprintf("Antrea:O:Drop:%s:", randomLogLabel)
+
+	// Before generating some traffic, there should be no corresponding kernel log on hostNetwork Pod x/a, and the kernel
+	// logs should not contain the generated random logLabel.
+	stdout, stderr, err := data.RunCommandFromPod(antreaNamespace, antreaPodName, agentContainerName, []string{"sh", "-c", "dmesg | tail -n 200"})
+	if err != nil || stderr != "" {
+		t.Fatalf("Got error: %v, %v", err, stderr)
+	}
+	require.NotContains(t, stdout, expectedLogPrefix)
+
+	// Generate some traffic that will be dropped by acnp-drop-x-to-y-egress. In this test, the target IP is the Antrea
+	// gateway IP of Node y.
+	_, err = k8sUtils.Probe(getNS("x"), "a", getNS("y"), "a", p80, ProtocolTCP, nil, nil)
+	require.NoError(t, err)
+
+	ipv4Enabled, ipv6Enabled := isIPv4Enabled(), isIPv6Enabled()
+
+	getNodeIPs := func(nodeName string) (string, string) {
+		nodeInfo := getNodeByName(nodeName)
+		require.NotNil(t, nodeInfo)
+		ip4, ip6 := nodeGatewayIPs(nodeInfo.idx)
+		if ipv4Enabled {
+			require.NotEmpty(t, ip4)
+		}
+		if ipv6Enabled {
+			require.NotEmpty(t, ip6)
+			ip6 = expandIPv6(ip6)
+		}
+		return ip4, ip6
+	}
+	srcIP4, srcIP6 := getNodeIPs(nodes["x"])
+	dstIP4, dstIP6 := getNodeIPs(nodes["y"])
+
+	// Add (?s) to match \n.
+	re4 := regexp.MustCompile(fmt.Sprintf(`(?s)%s.*SRC=%s DST=%s`, expectedLogPrefix, srcIP4, dstIP4))
+	re6 := regexp.MustCompile(fmt.Sprintf(`(?s)%s.*SRC=%s DST=%s`, expectedLogPrefix, srcIP6, dstIP6))
+
+	// TODO: Replace wait.PollUntilContextTimeout with assert.EventuallyWithT after https://github.com/stretchr/testify/issues/1396 is resolved.
```

Comment:
It seems the issue is already resolved, please check if you want to update in this PR or with a new PR to resolve this TODO.

Reply:
> It seems the issue is already resolved

Exactly, but the latest release of github.com/stretchr/testify is v1.9.0, which doesn't include the fix for the issue.